python/testapi/core/geometry/test_add_planes.py

```diff
@@ -30,7 +30,6 @@ def test_add_planes(self) -> None:
             brayns.Plane(4, 5, 6, 7)
         ])
         self.assertEqual(model.id, 0)
-        self.assertEqual(model.bounds, brayns.Bounds.empty)
-        self.assertEqual(model.metadata, {})
+        self.assertEqual(model.info, {})
         self.assertEqual(model.visible, True)
```

Comment:
Why removing the test of the bounds ?

Reply:
It is comparing it to bounds with min (0, 0, 0) and max (0, 0, 0). The rendered returned the same, but thats an error, since the must return infinite bounds to avoid interference with bound testings (i.e., the renderer now returns min(+inf), max(-inf) for invalid bounds, so the test was failing)